_episodes/07-errors.md

```diff
@@ -34,32 +34,28 @@ Let's examine one:
 ~~~
 # This code has an intentional error. You can type it directly or
 # use it for reference to understand the error message below.
-def favorite_ice_cream():
-    ice_creams = [
-        "chocolate",
-        "vanilla",
-        "strawberry"
-    ]
-    print(ice_creams[3])
-
-favorite_ice_cream()
+def lucky_dice(number):
+    outcomes=['win','lose']
+    if number >= 6:
+        print(outcomes[1])
+    if number < 6:
```

Comment:
else?

Reply:
Yes definitely "else". I am not sure this new example is clearer. I agree the initial example can be found a bit unrealistic but the main advantage is that it is fairly straightforward. I wonder if we would divert attendees by adding complexity while our main objective is to demonstrate how python tackles errors.